gneiss/regression/_transformer.py

```diff
@@ -0,0 +1,33 @@
+from ._ols import OLSModel
+from ._mixedlm import LMEModel
+from ._format import RegressionFormat_g
+from gneiss.plugin_setup import plugin
+
+
+@plugin.register_transformer
+def _1(data: OLSModel) -> RegressionFormat_g:
+    ff = RegressionFormat_g()
+    with ff.open() as fh:
+        data.write_pickle(fh)
+    return ff
```

Comment:
Can you open an [issue](https://github.com/biocore/biom-format/issues) with BIOM please? 

In this case, the file handle (`fh`) is closed by the context manager.